src/EnergyPlus/WeatherManager.hh

```diff
@@ -911,169 +914,29 @@ struct WeatherManagerData : BaseGlobalStruct
     Weather::AnnualMonthlyDryBulbWeatherData OADryBulbAverage; // processes outside air drybulb temperature
 
     // SetCurrentWeather static vars
-    int NextHour;
+    int NextHour = 1;
```

Comment:
I guess so.  I assume this will be handled by the environment manager so hopefully this initial value isn't actually important, but if it was already starting at 1 below, then this is fine.